### DISCUSSION TITLE
How to list sold products

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#17361] [How to]
Bit lost in understanding on how to list sold products. Manager want to see what has been sold at particular date, but all he can get is Goods Issued Note records where we cannot list line items as a column
Any pointer?

**Comment by ArniSamano** [comment]
It seems to be weird that this is not implemented. The modules surrounding inventory pointing out that we are on the edge of ERP, but without proper reporting it just impression

**Comment by TonyM15** [comment]
Agree with above. Non-existent proper reporting is a huge limitation in Yetiforce, Data analysis is a core of business, otherwise out of business.

**Comment by deepti4666** [comment]
Agree
